web_viewer/client/octree_viewer.ts

```diff
@@ -308,6 +309,27 @@ export class OctreeViewer {
       });
   }
 
+  public setMoving(value: boolean) {
+    if (value) {
+      for (const nodeId of Object.keys(this.loadedData)) {
+        if (
+          nodeId.length <= this.maxLevelToDisplay ||
+          this.loadedData[nodeId].threePoints === undefined
+        ) {
+          continue;
+        }
+        this.loadedData[nodeId].threePoints.visible = false;
+      }
+    } else {
+      for (const nodeId of Object.keys(this.loadedData)) {
```

Comment:
Since this looks like it's the exact same loop as in the branch above, I think it would be clearer to move the `if (value)` inside the loop.

Reply:
I would just rewrite the whole thing like this:

```
public setMoving(moving: boolean) {
  for (const nodeId of Object.keys(this.loadedData)) {
    const threePoints = this.loadedData[nodeId];
    if (threePoints != null) {
      // If we are moving, only show points above a certain depth. Otherwise, show them all.
      if (moving) {
        threePoints.visible = nodeId.length <= maxLevelToDisplay;
      } else {
        threePoints.visible = true;
      }
    }
  }
}
```

Reply:
that is very beautiful. I am buying this.